examples/nlp/huggingface_model/text_generation/llama/quantization/weight_only/main.py

```diff
@@ -17,33 +17,38 @@
 # pylint:disable=redefined-outer-name,logging-format-interpolation
 import os
 import onnx
+import time
 import json
 import random
 import torch
 import logging
 import argparse
+import random
 import numpy as np
-from datasets import load_dataset
+import datasets
 import onnxruntime as ort
-from torch.nn.functional import pad
-from torch.utils.data import DataLoader
-from intel_extension_for_transformers.transformers.llm.evaluation.lm_eval import evaluate
-from optimum.onnxruntime import ORTModelForCausalLM
-from transformers import LlamaConfig, LlamaTokenizer
-from neural_compressor_ort.quantization.calibrate import CalibrationDataReader
-
-logger = logging.getLogger(__name__)
-logging.basicConfig(format = "%(asctime)s - %(levelname)s - %(name)s -   %(message)s",
-                    datefmt = "%m/%d/%Y %H:%M:%S",
-                    level = logging.WARN)
+from transformers import LlamaTokenizer
+from transformers import LlamaConfig
+from torch.nn import functional
+from torch.utils import data
+from intel_extension_for_transformers.transformers.llm.evaluation import lm_eval
+from optimum import onnxruntime
```

Comment:
`onnxruntime` here is a bit confused with `import onnxruntime as ort`.
Can we directly `import optimum` and then call `optimum.onnxruntime.xxx`?